SPECS/mstflint/mstflint.spec

```diff
@@ -40,13 +56,33 @@ find . -type f -iname '*.cpp' -exec chmod a-x '{}' ';'
 %configure --enable-fw-mgr --enable-adb-generic-tools
 %make_build
 
+# Compile mstflint driver
+export EXTRA_CFLAGS='-DVERSION=\"%version\"'
+cd kernel
+set -- *
+mkdir source
+mv "$@" source/
+mkdir obj
+
+rm -rf obj
+cp -a source obj
+cd $PWD/obj
+make KPVER=%{kver}
+
 %install
 %make_install
 # Remove the devel files that we don't ship
 rm -fr %{buildroot}%{_includedir}
 find %{buildroot} -type f -name "*.la" -delete -print
 find %{buildroot} -type f -name '*.a' -delete
 
+export INSTALL_MOD_PATH=$RPM_BUILD_ROOT
```

Comment:
It looks like we might need to lint this and the other specs as we don't want to use `RPM_BUILD_ROOT`. [Here's a short doc](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/1126/Spec-linter) about configuring the one we use. The linter might also clean-up other details.

**NOTE**: the linter is not perfect, so I suggest running it after committing all previous changes and then double-checking the modifications it has made.